ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-missing-evpn-multicast-with-pim.yml

```diff
@@ -0,0 +1,48 @@
+type: l3leaf
+underlay_multicast: true
+# When this is set to false, the VRF won't have evpn_l3_multicast enabled
+# and PIM will fail
+evpn_multicast: false
+
+l3leaf:
+  defaults:
+    # LOOPBACK AND VTEP MANAGEMENT
+    loopback_ipv4_pool: 10.10.0.0/24
+    vtep_loopback_ipv4_pool: 10.11.0.0/24
+    vtep_loopback: Loopback1
+  nodes:
+    - name: failure-missing-evpn-multicast-with-pim
+      id: 3
+      bgp_as: 65101
+      filter:
+        tags: ['evpn-multicast-pegs']
+
+tenants:
+  - name: FABRIC
+    mac_vrf_vni_base: 10000
+    evpn_l3_multicast:
+      enabled: true
+      evpn_underlay_l3_multicast_group_ipv4_pool: 232.0.0.0/20
+    vrfs:
+      - name: Tenant_A_OP_Zone_MC
+        description: "Tenant_A_OP_Zone"
+        vrf_vni: 10
+        vtep_diagnostic:
+          loopback: 31
+          loopback_ip_range: 10.255.1.0/24
+        svis:
+          - id: 210
+            name: Tenant_A_OP_Zone_1
+            tags: ['evpn-multicast-l3']
+        l3_interfaces:
+          - nodes: [ failure-missing-evpn-multicast-with-pim ]
+            interfaces: [ Ethernet1 ]
+            ip_addresses: [ 10.254.248.0/31 ]
+            descriptions: [ "L3 Link to RPs" ]
+            enabled: true
+            pim:
+              enabled: true
+
+expected_error_message: >-
+  'pim: enabled' set on l3_interface 'Ethernet1' on 'failure-missing-evpn-multicast-with-pim'
+  requires 'evpn_multicast: true'
```

Comment:
```suggestion
  requires 'evpn_multicast: true' to be set in group vars.
```

Reply:
well technically you could set it in host_vars and it would still work if only this device has a l3interface so it is a bit misleading